ajax/libs/angular-pageslide-directive/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "angular-pageslide-directive",
+  "filename": "angular-pageslide-directive.min.js",
+  "author": "Daniele Piccone <mail@danielepiccone.com>",
+  "description": "AngularJS sliding panel for serving additional content from off the page",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:dpiccone/ng-pageslide.git"
```

Comment:
Let's modify to `git://github.com/dpiccone/ng-pageslide.git`
We hope this link can be used without [SSH keys setting](https://github.com/settings/keys) to clone the repo.
Many thanks.

Reply:
The url has changed in v2.1.5 since I had to change my github username https://github.com/danielepiccone/ng-pageslide/blob/v2.1.5/package.json